### PR TITLE
Added a `mpl-toolbox` section to the Umi docs

### PR DESCRIPTION
**Pull request by L0STE**
- Added a `mpl-toolbox` section to the Umi docs (in the introduction part)
- Deleted the `mpl-toolbox` product section since it was redundant
- Added redirect for all the `mpl-toolbox` product section to point to "/umi/toolbox"
- Changed the `middleware.js` file to make it cleaner for different roots + handle empty sub-roots (like /toolbox)

### CODE REVIEW

**Comment by L0STE** [comment]
Update: now the toolbox is on the side, and live as it's own "label" and not in the introduction as previously said

**Comment by tonyboylehub** [comment]
Few things, you have double headings on the start of each page.

secondly I would not have split these into programs but individual areas

This all still feels hidden behind terms people are not going to know where to look.

Can we break out functions into invidual pages for some things

**ToolBox**
- Priority Fees and Compute Mangement (Compute Budget Program)
- Create Account (Create Account and Create Account with rent)
- Transfer Sol (Transfer Sol and Transfer All Sol)
- SPL Token Management (Token Program, Associated Token Program & Extras)
- Address Lookup Tables
- Transaction Memo

File: src/pages/umi/toolbox/transfer-sol.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+The following instructions are part of the System Program and MPL System Extras Program. You can learn more about the System Program in [Solana's official documentation](https://docs.solanalabs.com/runtime/programs#system-program).
+
+## Transfer SOL
+
+This instruction allows you to transfer SOL from one account to another. You need to specify the amount of SOL to transfer in **lamports** (1/1,000,000 of SOL)
+
+```ts
+import { sol, publicKey } from '@metaplex-foundation/umi'
+import { transferSol, transferAllSol } from '@metaplex-foundation/mpl-toolbox'
```
Comment (tonyboylehub) [inline src/pages/umi/toolbox/transfer-sol.md:15]: Unused import

```suggestion
import { transferSol } from '@metaplex-foundation/mpl-toolbox'
```

**Comment by tonyboylehub** [comment]
This looks much better now.

**Review by tonyboylehub — APPROVED** [review]

**Pull request merged**